src/components/annual-target-hours/accruals/AnnualTargetHours.jsx

```diff
@@ -0,0 +1,104 @@
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import '../../../i18n';
+
+const AnnualTargetHours = ({ targetData, accrualsData }) => {
+  const { t } = useTranslation('common');
+  const total = targetData?.targetTotal || 0;
+  const worked = accrualsData?.cumulativeTotal || 0;
+  const remainingMins = Math.floor(total - worked);
+  const remainingWhole = Math.floor(remainingMins / 60);
+  const target = total - (accrualsData?.cumulativeTarget || 0);
+
+  return (
+    <div className="accruals-container">
+      <form className="grey-border">
+        <h1 className="govuk-heading-m newline">
+          {targetData ? (
+            <span
+              dangerouslySetInnerHTML={{
+                __html: t('annualTargetHours.titleRemaining', {
```

Comment:
NIT: Could we rename titleRemaining to remainingHoursTitle?

Reply:
Renamed to more suitable variable name